client/app/account/billing/autoRenew/update/billing-autoRenew-update.html

```diff
@@ -25,6 +25,17 @@
                                     <td class="oui-datagrid__cell">
                                         <span data-ng-bind-html="'autorenew_service_type_'+ service.serviceType | translate"></span>
                                     </td>
+                                    <td class="oui-datagrid__cell">
+                                        <select name="renewPeriod"
```

Comment:
This can be replaced by a oui-select no ?

Reply:
Thanks for the comment. The ui-kit classes has been added.